src/action/bandaidset.rs

```diff
@@ -231,8 +231,8 @@ pub(crate) mod tests {
                     .to_owned(),
             },
             BandAid {
-                span: (2_usize, 3..43).try_into().unwrap(),
-                replacement: " automate that. Maybe not. But writing this is annoying".to_owned(),
+                span: (2_usize, 0..43).try_into().unwrap(),
+                replacement: "/// automate that. Maybe not. But writing this is annoying".to_owned(),
```

Comment:
but why would we replace the `///` ? we want to preserve them, no? That way we become invariant to the comment type.

Reply:
At some point, we do have to add `///` for additional lines. In my opinion, it was more concise to add `/// ` on all multiline replacements rather than checking if it is a newly added line. 

However, if we took the approach of splitting `BandAid` into an enum, the above approach will be better. As this option is also a good choice for dealing with deletion/addition of lines in general, I suggest I'll start implementing it.

Reply:
> At some point, we do have to add `///` for additional lines. In my opinion, it was more concise to add `/// ` on all multiline replacements rather than checking if it is a newly added line.

My point is rather, that we don't necessarily. We only need to add additional lines for `///`, but not necessarily for `#[doc=".."]` (depending on the discussion above).

> However, if we took the approach of splitting `BandAid` into an enum, the above approach will be better. As this option is also a good choice for dealing with deletion/addition of lines in general, I suggest I'll start implementing it.

:+1:

Reply:
In any case, I think its much simpler to refactor `BandAid` :)